apps/coscrad-frontend/src/components/nav-menu/nav-menu-presenter.tsx

```diff
@@ -22,7 +22,7 @@ export const NavMenuPresenter = ({ navItemInfos }: NavMenuPresenterProps): JSX.E
     };
 
     return (
-        <Box data-testid="nav-menu">
+        <Box sx={{ display: 'flex' }} data-testid="nav-menu">
```

Comment:
The icons stacked on top of each other on mobile screens if our site title overflowed in to two lines. Adding display: flex allows the icons to flex horizontally.